stryker/angular/README.md

```diff
@@ -0,0 +1,41 @@
+# Angular
+Stryker supports Angular projects using the Angular CLI starting from @angular/cli v6.1.0. Are you using an older version? Then there are some tips later in this document.
+
+## @angular/cli 6.1.0 and higher
+Recommended stryker packages: `npm i -D stryker-api stryker stryker-karma-runner stryker-typescript stryker-html-reporter`
+
+### Configuration
+After installing the recommended packages, create the `stryker.conf.js` file in your repository. 
+The configuration below contains a good starting point for Angular projects. 
+You may have to change some paths or config settings like the selected browsers. 
+We highly suggest using a headless browser when testing using stryker.
+
+Coverage analysis with [stryker-jasmine](http://npmjs.com/package/stryker-jasmine) is unfortunately not supported as of right now.
+```js
+module.exports = function (config) {
```

Comment:
This sample config will save people lots of time, good job!

Reply:
How can we scrape this sample during `stryker init`?

Reply:
As of right now, we can't. We can always refactor these documents later so the code could also be used during stryker init.